t/01-test_needle.t

```diff
@@ -94,27 +94,6 @@ $needle = needle->new('kde.ref.json');
 $res = $img1->search($needle);
 ok(!defined $res, "no match with different art");
 
-subtest 'handle failure to load image' => sub {
```

Comment:
Removing this whole subtest looks wrong.

Reply:
Thanks for this hint! I was able to keep most parts.